packages/core/src/builder.ts

```diff
@@ -459,7 +459,7 @@ export default class SchemaBuilder<Types extends SchemaTypes> {
     const values =
       typeof param === 'object'
         ? // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
-          valuesFromEnum<Types>(param as BaseEnum)
+          valuesFromEnum<Types>(param as BaseEnum, options?.values as any)
```

Comment:
We general try to avoid any where ever possible in Pothos, casting to the type expected by the function should work (not 100% sure, there might be a comparability issue), you could cast through unknown, or cast to never if you really need to, but linting (pnpm lint) won't pass with the any